app/models/concerns/hud_reports/clients.rb

```diff
@@ -37,10 +37,16 @@ module HudReports::Clients
     # and the reporting period end is more than a year since the beginning of the enrollment
     # and the enrollment started more than one year ago
     private def annual_assessment_expected?(enrollment)
-      return false unless enrollment.head_of_household?
+      return false unless enrollment.present? && enrollment.head_of_household?
 
       end_date = [enrollment.last_date_in_program, report_end_date, Date.current].compact.min
-      enrollment.first_date_in_program + 1.years < end_date
+      enough_days = if enrollment.project.bed_night_tracking?
+        enrollment.enrollment.services.bed_night.count > 365
```

Comment:
Do these services need to have occurred before the end of the reporting period?

Reply:
well, really before `end_date`

Reply:
Good catch! Yes.